src/player.cpp

```diff
@@ -10102,6 +10102,20 @@ int player::get_wind_resistance(body_part bp) const
     return totalCoverage;
 }
 
+int player::get_weight() const
```

Comment:
There already is a get_weight function in character class, which doesn't take inventory into account. This one is specific to the player.